gst/tensor_aggregator/tensor_aggregator.c

```diff
@@ -167,6 +178,12 @@ gst_tensor_aggregator_class_init (GstTensorAggregatorClass * klass)
           "The number of frames to flush (0 to flush all output)", 0, G_MAXUINT,
           DEFAULT_FRAMES_FLUSH, G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 
+  g_object_class_install_property (object_class, PROP_FRAMES_DIMENSION,
+      g_param_spec_int ("frames-dim", "Frames dimension",
+          "The index of frames in tensor dimension (default value -1 does not change the dimension)",
+          -1, (NNS_TENSOR_RANK_LIMIT - 1), DEFAULT_FRAMES_DIMENSION,
+          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
```

Comment:
Do you mean:
```
"The dimension index of frames in tensor (default value -1 means that it does not have multiple frames)"
```
Anyway, is this for "input tensor"?

Please add a commit that describes the relation between input/output (esp. dimensions) with the values of frames-in, frames-out, frames-flush, frames-dim as comments in this file. Or please add ```gst/tensor_aggregator/README.md``` and describe there as a user manual; it is too difficult to understand options. You may use Project:NS case as the example.

Reply:
Updated a description for the properties.